crates/oxc_ast/src/ast/js.rs

```diff
@@ -1,6 +1,6 @@
-// NB: `#[span]`, `#[scope(...)]`, `#[visit(...)]`, `#[visit_as(...)]` and `#[visit_args(...)]` do
-// not do anything to the code, They are purely markers for codegen used in
-// `tasts/ast_codegen` and `crates/oxc_traverse/scripts`. See docs in that crate.
+// NB: `#[span]`, `#[scope(...)]` and `#[visit(...)]` do NOT do anything to the code,
+// They are purely markers for codegen used in
+// `tasts/ast_codegen` and `crates/oxc_traverse/scripts`. See docs in those crates.
```

Comment:
Small nits:

1. Typo: `tasks` not `tasts`.
2. Can we add `#[ast]` to the list of attrs which don't do anything in this comment?
3. Full stop not comma after "do NOT do anything to the code" (that really is a micro-nit!)

Reply:
> 2. Can we add #[ast] to the list of attrs which don't do anything in this comment?

We can but it would be removed in the next PR, We actually do something now in our `ast` attribute.

I'll fix the other 2, Let me know what are your thoughts on this one.

Reply:
Ok cool, feel free to leave out that one if you prefer. We'll need to put a longer comment on the files once we do the whole `#[repr(C)]` thing, to explain what `#[ast]` does.